integrated_channels/canvas/client.py

```diff
@@ -240,7 +240,7 @@ def _delete(self, url):
         Args:
             url (str): The canvas url to send delete requests to.
```

Comment:
Can you add a note to the doc string that we're concluding the course and not deleting it?